src/start-client.jsx

```diff
@@ -4,6 +4,7 @@ import React from 'react';
 import { hydrate } from 'react-dom';
 import { Provider } from 'react-redux';
 import { IntlProvider } from 'react-intl-redux';
+import cookie from 'react-cookie';
```

Comment:
old code, should be updated to universal-cookie

Reply:
ah, I see https://github.com/plone/volto/blob/master/CHANGELOG.md#breaking-1

Reply:
✅